### DISCUSSION TITLE
Removing 1px dark mode border from windows

### DISCUSSION THREAD

**Discussion by oschrenk** [asmvik/yabai#2024] [General]
I thought the yabai community might know best. 

Is there a way to remove the 1px white border around the windows In dark mode? I'm not really looking for any new window manager but only for removing the border. 

Would yabai help with that? Is there a "standalone" app that can do that?

**Comment by hamadahiro** [comment]
Turning off window_shadow should remove that 1x white border, but after some time, it returns to some windows until you restart Yabai again.

My config is set to:

`window_shadow float`

so that only the floating windows on top have shadows (and therefore 1px white border)

Here's the difference with window_shadow turned on and off/float.

![Screenshot 2023-12-31 at 5 13 28 PM](https://github.com/koekeishiya/yabai/assets/149450672/853e53ef-ac40-4045-abde-50887bcb5d78)

![Screenshot 2023-12-31 at 5 12 55 PM](https://github.com/koekeishiya/yabai/assets/149450672/96ad3297-5300-404b-ba96-99b26c4719a7)
